utils/log-target.js

```diff
@@ -30,10 +30,15 @@ const CDN_REGEX = new RegExp(
 module.exports = class LoggingTarget {
   constructor(referrer, reportingParams) {
     this.opts = { referrer, ...reportingParams };
+    this.log = null;
+
+    this.ready = this.logPromise().then(log => {
```

Comment:
Nit: Why not just return the `logPromise`?

```js
// before
await logTarget.ready;
const log = logTarget.log;

// After
const log = await logTarget.log;

Reply:
Done.